Python/Product/EnvironmentsList/PipExtensionProvider.cs

```diff
@@ -143,6 +143,11 @@ public sealed class PipExtensionProvider : IEnvironmentViewExtension {
         internal async Task<IList<PipPackageView>> GetInstalledPackagesAsync() {
             string[] args;
 
+            if (!CanExecute) {
+                // Invalid configuration, so assume no packages

```

Comment:
Yea doing it at Create seems best.  I guess we can't really tell why we can't execute here.  Although maybe CanRun could be the one to output the message.  Anyway I've made an issue to track doing something better to prevent bad environments (#790)